archivers/lzma/Portfile

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem      1.0
+PortGroup       compiler_blacklist_versions 1.0
 PortGroup       legacysupport 1.1
 PortGroup       makefile 1.0
 
 name            lzma
 version         22.01
-revision        0
+revision        1
```

Comment:
it looks like this change is made to fix build failures, in that case there is no reason to increase the `revision`